numba_dpex/core/dpex_exceptions.py

```diff
@@ -32,3 +33,164 @@ def __init__(self, kernel_name, return_type) -> None:
         )
 
         super().__init__(self.message)
+
+
+class InvalidKernelLaunchArgsError(Exception):
+    """Raised when a kernel is dispatched with insufficient or incorrect launch
```

Comment:
```suggestion
    """Exception raised when a kernel is dispatched with insufficient or incorrect launch
```

to be consistent with docstring used for `KernelHasReturnValueError`.